### PR TITLE
xcode-install: init at 2.6.8

### PR DESCRIPTION
**Pull request by q3k**
###### Motivation for this change

Add xcode-install, a Ruby tool to manage globally installed Xcode versions on macOS systems.

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [X] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [X] Ensured that relevant documentation is up to date
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/tools/xcode-install/default.nix [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{ bundlerApp }:
```
Comment (siraben) [inline pkgs/development/tools/xcode-install/default.nix:1]:
```suggestion
{ lib, bundlerApp }:
```

`lib` is not in scope.

**Review by siraben — CHANGES_REQUESTED** [review]

File: pkgs/development/tools/xcode-install/default.nix (hunk 1 above)
Comment (q3k) [reply pkgs/development/tools/xcode-install/default.nix:1]: Ah, you're right, didn't change this after adding the metadata. Whoops!

**Comment by siraben** [comment]
Result of `nixpkgs-review pr 110923` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>xcode-install</li>
  </ul>
</details>

**Review by siraben — APPROVED** [review]
LGTM. Builds and runs.

**Pull request merged**